### ISSUE TITLE
Allow click-through from mutant to unit test and reverse

### ISSUE THREAD

**Issue by rouke-broersma** [stryker-mutator/mutation-testing-elements#2490]
It would be nice to be able to click-through from a mutant to a covering unit test or from a unit test to the mutants it covers.

From:

![image](https://user-images.githubusercontent.com/3595932/236497389-2e6436a6-68c6-474c-9d2a-08354380fbb5.png)

To:

![image](https://user-images.githubusercontent.com/3595932/236497508-67cf1da1-8303-4eac-bc6f-11654ac8a6cc.png)

And back

**Comment by nicojs** [comment]
Yes, i think this is pretty straightforward to add. I would suggest adding a line number and focussed mutant/test to the url fragment.
